.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we really need this file?

Reply:
Well don't know tbh, I just added the new language via Xcode and this was automatically generated with it. Feel free to remove it, if you think it isn't necessary. The only thing I need is just the Czech localisation.

Reply:
done, can we merge now? 🙂